server/src/instant/model/app.clj

```diff
@@ -36,6 +36,38 @@
 (defn get-by-id! [params]
   (ex/assert-record! (get-by-id params) :app {:args [params]}))
 
+(defn list-account-apps
+  ([user-id] (list-account-apps aurora/conn-pool user-id))
+  ([conn user-id]
+   (sql/select conn
+               ["SELECT a.*
+                FROM apps a
+                WHERE a.creator_id = ?::uuid"
+                user-id])))
+
+(comment
+  (def u "6412d553-2749-4f52-898a-0b3ec42ffd28")
+  (list-account-apps u))
+
+(defn get-account-app-by-id
```

Comment:
nit: maybe would call this `get-one-by-creator-id`. Reasoning: right now it sounds like it is returning an `account`, but it's actually 'get one app by id and creator'

Reply:
what about `get-creator-app-by-id`? basically i want "get-app-by-id" for a specific creator... do you still like `get-one-by-creator-id` for this?

Reply:
Maybe 

`get-by-creator-and-app-id`? 

Admittedly it is looking verbose

Right now the heuristic is something like 

`get-{name-of-model, but removed for brevity}-{qualifier}`

For example, there's a `get-by-id` in there. 

cc @dwwoelfel if you have thoughts too!

Reply:
how about... `get-by-id-and-creator`? 😛

Reply:
Love it!